.github/workflows/linting.yml

```
@@ -0,0 +1,43 @@
+name: Linting
+
+on:
+  push:
+    branches: [ sj/ga-migrate-linting ]
+
+  
+jobs:
+  linting:
+    runs-on: ubuntu-latest
```

I think it's better to fix the ubuntu version. 22.04 is not that far away 😬 
```suggestion
    runs-on: ubuntu-20.04
```